processing/data_collection/gazette/pipelines.py

```diff
@@ -22,6 +22,21 @@ def process_item(self, item, spider):
         return item
 
 
+class DefaultValuesPipeline:
```

Comment:
**praise**: I liked this pipeline very much. It doesn't make sense to add extra lines to the spider with values that can be included automatically.